Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.6
+
+WORKDIR /
+COPY . /
```

Comment:
If we are copying the entire directory then we should probably add a `.dockerignore` file.
This will reduce the amount we copy into the container. see https://github.com/ONSdigital/eq-survey-runner/blob/master/.dockerignore as an example.